app/src/main/java/com/example/playlistmaker/Track.kt

```diff
@@ -11,6 +11,7 @@ data class Track(
     val collectionName: String,
     val releaseDate: String,
     val primaryGenreName: String,
-    val country: String
+    val country: String,
+    val previewUrl: String
```

Comment:
```suggestion
    val previewUrl: String,
```
В котлине можно добавлять `trailing comma`, какие преимущества это даёт можно узнать здесь - https://kotlinlang.org/docs/coding-conventions.html#trailing-commas